production/helm/loki/templates/service-headless.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
what is the headless service for in this repository? Are there any pods communicating over this service?

Reply:
As mentioned in the description, it's required for a `StatefulSet`. Also, it provides a stable hostname which allows consul to be used as a ring store.